services/user.py

```diff
@@ -0,0 +1,51 @@
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> None:
+    user = User.objects.create_user(
+        username=username,
+        password=password
+    )
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
+
+
+def update_user(
+    user_id: int,
+    username: str = None,
+    password: str = None,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> None:
+    user = User.objects.get(id=user_id)
```

Comment:
Similar to the previous comment, using `get` without handling exceptions can lead to `DoesNotExist` errors if the user ID is not found. Consider handling this exception to avoid runtime errors.